software/requirements.txt

```diff
@@ -0,0 +1,8 @@
+inputs==0.5
+numpy==1.21.3
+opencv_python==4.6.0.66
+pandas==1.5.3
+pyrealsense2==2.53.1.4623
+pyserial==3.5
+segment==0.2.0
```

Comment:
`segment` is from [boot camp code](https://github.com/ut-robotics/picr22-boot-camp-programming), not this package: https://pypi.org/project/segment/.
It can't be installed with pip.

Reply:
My bad, I didn't see it after running the pipreqs command to generate the requirements.txt... It's fixed now

Reply:
`segment` module from boot camp code still needs installation. Instructions should be added to README.

Reply:
I've added a few lines to the documentation about this module. Please let me know if it works with you.

Reply:
After cloning you can follow the installation instructions in the [boot camp code README](https://github.com/ut-robotics/picr22-boot-camp-programming/blob/main/README.md):
> How to use
>Segmentation module installation:
>```
>cd segment_module
>pip3.9 install .
>```

There is no need to move the files around.

Reply:
Is it better now?